src/main/java/dinkplugin/domain/PlayerLookupService.java

```diff
@@ -0,0 +1,36 @@
+package dinkplugin.domain;
+
+public enum PlayerLookupService {
+    NONE("None"),
+    OSRS_HISCORE("OSRS HiScore"),
+    CRYSTAL_MATH_LABS("Crystal Math Labs"),
+    TEMPLEOSRS("Temple OSRS"),
+    WISEOLDMAN("Wise Old Man");
+
+    private final String name;
+
+    PlayerLookupService(String name) {
+        this.name = name;
+    }
+
+    public String playerUrl(String playerName) {
+        switch (this) {
+            case OSRS_HISCORE:
+                return "https://secure.runescape.com/m=hiscore_oldschool/hiscorepersonal.ws?user1=" + playerName;
```

Comment:
```suggestion
                return "https://secure.runescape.com/m=hiscore_oldschool/hiscorepersonal?user1=" + playerName;
```
I lied, it can be fixed